build-tools/packages/build-cli/docs/generate.md

```diff
@@ -11,6 +11,8 @@ Generate commands are used to create/update code, docs, readmes, etc.
 * [`flub generate entrypoints`](#flub-generate-entrypoints)
 * [`flub generate packlist`](#flub-generate-packlist)
 * [`flub generate releaseNotes`](#flub-generate-releasenotes)
+* [`flub generate source-entrypoints`](#flub-generate-source-entrypoints)
+* [`flub generate sourceEntrypoints`](#flub-generate-sourceentrypoints)
```

Comment:
We should only have one of these. Looks like there is a pre-existing pattern for `sourceEntrypoints`.